### DISCUSSION TITLE
Is it possible to use smooth shading?

### DISCUSSION THREAD

**Discussion by GuillaumeFavelier** [pyvista/pyvista#2209] [Q&A]
In the examples I noticed the surfaces use flat shading by default and the [smooth function](https://docs.pyvista.org/types/points.html?highlight=smooth#pyvista.PolyData.smooth) I found in the API although it “relaxes” the mesh, it's not exactly what I'm looking for. Do you have any plans to add such a feature?

![image](https://user-images.githubusercontent.com/18143289/58173198-065c5400-7c9b-11e9-9156-1f6dc912ae6f.png)

**Comment by akaszynski** [comment]
Correct, the smooth function in the API only smooths the mesh and doesn't actually change the way the mesh is displayed.  I'd like to implement mesh smoothing as well, and right now ``pyvista`` doesn't have that feature.

**Comment by GuillaumeFavelier** [comment]
I shared a version of the code in [#230](https://github.com/pyvista/pyvista/pull/230) where I got something promising I think for a given `vtk.vtkPolyData` at least.

**Comment by banesullivan** [comment]
Considering https://github.com/pyvista/pyvista/pull/230 is merged, is this issue resolved? @GuillaumeFavelier, could you perhaps add an example to the gallery to demonstrate the different types of shading?

**Comment by GuillaumeFavelier** [comment]
Yes it is resolved! And I would be glad to demonstrate this feature with an example.
